tasks/locallink.go

```diff
@@ -0,0 +1,26 @@
+package tasks
+
+import (
+	"context"
+	"fmt"
+
+	ipld "github.com/ipld/go-ipld-prime"
+)
+
+type localLink struct {
+	ipld.Node
+}
+
+var _ ipld.Link = (*localLink)(nil)
+
+func (l *localLink) LinkBuilder() ipld.LinkBuilder {
```

Comment:
Just a heads up this is all going to need to be rewritten if we switch to LinkSystem

Reply:
hmm. this actually won't work because of https://github.com/ipld/go-ipld-prime/blob/master/codec/dagjson/marshal.go#L191 :-|

Reply:
ok. i ended up making my own codec for the purposes of json-ing in the way I want and got thwarted again by the great ipld 0.7/0.9 fissure, but i think this logs in the way i want now :)